lute/std/libs/test.luau

```diff
@@ -61,6 +61,7 @@ type FailedTest = {
 	error: string, -- the error message from the assertion
 	filename: string, -- source file where the failure occurred
 	linenumber: number, -- line number of the failure
+	stacktrace: string, -- full stacktrace at the point of failure
```

Comment:
If you look at `tests/cli/test/test.test.luau` (sorry for horrific naming lol), can you add a test for this? Moreover is it possible to trim the stacktrace? E.g. show the last 5 frames before the crash?